go-controller/pkg/ovn/controller/egress_services/egress_services_node.go

```diff
@@ -198,21 +207,10 @@ func (c *Controller) syncNode(key string) error {
 			}
 			delete(c.nodes, nodeName)
 			state.healthClient.Disconnect()
-		} else {
-			// we don't have a node at this point (node deleted?) and we don't have its cache
-			// entry (state==nil) as well. Maybe state was deleted when node became nodeReady or unreachable
-			// nothing to sync here
-			return nil
 		}
 
-		return c.deleteNoRerouteNodePolicies(c.addressSetFactory, nodeName, state.v4InternalNodeIP,
```

Comment:
Is `v4InternalNodeIP/v6InternalNodeIP` used anywhere else in egress services? If it is not we can remove it from the state struct